portingdb/htmlreport.py

```diff
@@ -158,6 +168,15 @@ def get_groups(db, query):
     return groups
 
 
+def get_naming_progress_info(correct_count, misnamed_count):
+    NameStatus = namedtuple('NameStatus', 'name color term url description')
```

Comment:
Please define classes outside of functions, unless there's a reason not to do it.